POSHIT/Example CHT application/malaria-usecase-cht/nools-extras.js

```diff
@@ -1,13 +1,72 @@
+const { DateTime } = require('luxon');
+const { FORMS } = require('./shared-extras');
+const { getField, isFirstReportNewer } = require('cht-nootils')();
+
 const isFormArraySubmittedInWindow = (reports, formsArray, startTime, endTime) => {
```

Comment:
This function exists in cht-nootils. I would recommend reusing it unless you have a very specific need to override some behaviour

Reply:
@brian-gacheru , you have maintained this. Are you using it to override the function in cht-nootils?

Reply:
Yes this function overrides the one in `cht-nootils`. Using the one in `cht-nootils` crashes the app